src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media/GeometryTests/UnoSamples_Tests.ClosedFigurePage.cs

```diff
@@ -25,7 +25,7 @@ public void ClosedPath()
 
 			using var screenshot = TakeScreenshot("Closed state");
 
-			ImageAssert.HasColorAt(screenshot, 50, 300, Color.Black);
+			ImageAssert.HasColorAt(screenshot, 25, 300, Color.Black);
```

Comment:
Is it worth having an explanation as to why this position was chosen to test the color (and why 50,300 wasn't good?)

Reply:
Reverted